[2025-03-22T18:23:32.307+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-22T18:23:32.316+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag.first_task manual__2025-03-22T18:23:31.099468+00:00 [queued]>
[2025-03-22T18:23:32.319+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag.first_task manual__2025-03-22T18:23:31.099468+00:00 [queued]>
[2025-03-22T18:23:32.320+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-03-22T18:23:32.326+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): first_task> on 2025-03-22 18:23:31.099468+00:00
[2025-03-22T18:23:32.331+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag', 'first_task', 'manual__2025-03-22T18:23:31.099468+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmp4iqo95z8']
[2025-03-22T18:23:32.333+0000] {standard_task_runner.py:105} INFO - Job 4: Subtask first_task
[2025-03-22T18:23:32.333+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=591) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-22T18:23:32.333+0000] {standard_task_runner.py:72} INFO - Started process 592 to run task
[2025-03-22T18:23:32.358+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag.first_task manual__2025-03-22T18:23:31.099468+00:00 [running]> on host 4f1e28758fdd
[2025-03-22T18:23:32.399+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='our_first_dag' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-22T18:23:31.099468+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-22T18:23:31.099468+00:00'
[2025-03-22T18:23:32.400+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-22T18:23:32.409+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-22T18:23:32.411+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo Hello World!']
[2025-03-22T18:23:32.417+0000] {subprocess.py:99} INFO - Output:
[2025-03-22T18:23:32.419+0000] {subprocess.py:106} INFO - Hello World!
[2025-03-22T18:23:32.419+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-22T18:23:32.431+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-22T18:23:32.431+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=our_first_dag, task_id=first_task, run_id=manual__2025-03-22T18:23:31.099468+00:00, execution_date=20250322T182331, start_date=20250322T182332, end_date=20250322T182332
[2025-03-22T18:23:32.475+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-22T18:23:32.486+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-22T18:23:32.487+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
